portal/scripts/generateCodesplit.mjs

```diff
@@ -15,16 +24,33 @@ for (const key of Object.keys(packageJSON["dependencies"])) {
 }
 deps.sort();
 
+const productionOnlyDeps = [
+  "@apollo/client",
+  "@fluentui/react",
+  "@fluentui/react-hooks",
+  "@oursky/react-messageformat",
+  "react-dom",
+  "react-helmet-async",
+  "react-router-dom",
+];
+
 const filehandle = await open("./src/codesplit.ts", "w");
 for (const dep of deps) {
   // Special case: normalize.css is referenced by index.html.
-  if (dep !== "normalize.css") {
-    await filehandle.write("import(");
-    await filehandle.write(JSON.stringify(dep));
-    // We include whitespaces here so that the generated file is formatted
-    // according to Prettier's taste.
-    // So npm run fmt WILL NOT format that file again.
-    await filehandle.write(").finally(() => {});\n");
+  if (dep !== "normalize.css" && !productionOnlyDeps.includes(dep)) {
+    await writeImport(dep);
   }
 }
+
+await filehandle.write("\n");
+// We add this line to ignore "'process' is not defined" eslint warning
```

Comment:
Ignore this file in [.eslintignore](https://eslint.org/docs/latest/user-guide/configuring/ignoring-code)